### ISSUE TITLE
Add a imv.desktop file

### ISSUE THREAD

**Issue by guiniol** [eXeC64/imv#15]
So that it can be set as default for xdg-open etc...

Based on the feh one, this is what it could look like. The Icon is probably not necessary for tiling WM.

```
[Desktop Entry]
Name=IMV
GenericName=Image viewer
GenericName[en_US]=Image viewer
Comment=Fast freeimage-based Image Viewer
Exec=imv %F
Terminal=false
Type=Application
Icon=/path/to/icon.png
Categories=Graphics;2DGraphics;Viewer;
MimeType=image/bmp;image/gif;image/jpeg;image/jpg;image/pjpeg;image/png;image/tiff;image/x-bmp;image/x-pcx;image/x-png;image/x-portable-anymap;image/x-portable-bitmap;image/x-portable-graymap;image/x-portable-pixmap;image/x-tga;image/x-xbitmap;
Name[en_US]=imv
```

**Comment by eXeC64** [comment]
Good idea, thanks.

**Comment by guiniol** [comment]
If the above entry without the Icon is fine with you, I can do a PR later (probably tonight)

**Comment by guiniol** [comment]
Actually, it's in #16, comments are welcome.

**Comment by eXeC64** [comment]
Closed as duplicate of #16.
